.env.example

```diff
@@ -3,3 +3,4 @@ DESCOPE_FLOW_ID="sign-up-or-in" # Your Descope flow ID
 DESCOPE_FLOW_DEBUG= # Set to true in case you want to debug your flow
 REACT_APP_DESCOPE_BASE_URL= # Descope API base URL
 REACT_APP_USE_ORIGIN_BASE_URL= # Set in case you want to use the origin as
+DESCOPE_TENANT_ID= # Descope tenant ID
```

Comment:
I suggest to have it after the `DESCOPE_FLOW_ID`

Reply:
Done